android/modules/media/src/java/ti/modules/titanium/media/MediaModule.java

```diff
@@ -910,12 +912,22 @@ public void openPhotoGallery(KrollDict options)
 		TiActivitySupport activitySupport = (TiActivitySupport) activity;
 
 		TiIntentWrapper galleryIntent = new TiIntentWrapper(new Intent());
-		galleryIntent.getIntent().setAction(Intent.ACTION_PICK);
+		galleryIntent.getIntent().setAction(Intent.ACTION_GET_CONTENT);
```

Comment:
btw. this is not a necessary change but recommended by Google since `ACTION_PICK` can be considered as deprecated (http://stackoverflow.com/questions/6486716/using-intent-action-pick-for-specific-path/6486827#6486827)